docs/reference/modules/threadpool.asciidoc

```diff
@@ -142,14 +142,23 @@ threadpool:
 [[processors]]
 === Processors setting
 The number of processors is automatically detected, and the thread pool
-settings are automatically set based on it. Sometimes, the number of processors
-are wrongly detected, in such cases, the number of processors can be
-explicitly set using the `processors` setting.
+settings are automatically set based on it. However, the number of
```

Comment:
I've seen both, I think I'd agree with you that the case you mention is more common. I will add a note.